client/src/components/MultiJurisdictionAudit/RoundManagement/generateSheets.ts

```diff
@@ -2,42 +2,55 @@ import jsPDF from 'jspdf'
 import { IAuditBoard } from '../useAuditBoards'
 import { IBallot } from './useBallots'
 
+// Label constants in points
```

Comment:
accidentally measured these in points instead of millimeters and didnt feel like going back to remeasure